package.json

```diff
@@ -55,6 +58,7 @@
     "plop": "^2.0.0",
     "prettier": "^1.14.0",
     "react": "^16.8.1",
+    "react-apollo": "^2.5.2",
```

Comment:
these new packages added to `devDependencies` are just to squelch some peer dependency warnings.